src/main/java/core/basesyntax/services/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.services;
+
+import core.basesyntax.figures.*;
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure(){
+        ColorSupplier service = new ColorSupplier();
+        String color = service.getRandomColor();
+
+        Random rand = new Random();
+
+        Shape[] figures = Shape.values();
+        String figure = String.valueOf(figures[rand.nextInt(figures.length)]).toLowerCase();
+
+        int firstLength = rand.nextInt(100);
+        int secondLength = rand.nextInt(100);
+        int thirdLength = rand.nextInt(100);
```

Comment:
Please define the range for random numbers as constants for better readability and maintainability.

Reply:
Great advice! Thank you